internal/evaluator/conftest_evaluator.go

```diff
@@ -279,6 +279,18 @@ func (c conftestEvaluator) Evaluate(ctx context.Context, inputs []string) (Check
 	return results, nil
 }
 
+// Helper function to sort Result lists.
```

Comment:
Nitpick, seems like it will get refactored regardless. The convention is to start with the function name (see https://go.dev/blog/godoc)

```suggestion
// SortResults sorts Result slices.
```

Reply:
Updated, thanks!